src/erlcloud_aws.erl

```diff
@@ -816,6 +816,19 @@ timestamp_to_gregorian_seconds(Timestamp) ->
     {ok, [Yr, Mo, Da, H, M, S], []} = io_lib:fread("~d-~d-~dT~d:~d:~dZ", binary_to_list(Timestamp)),
     calendar:datetime_to_gregorian_seconds({{Yr, Mo, Da}, {H, M, S}}).
 
+-spec region(aws_config()) -> {ok, string()} | {error, term()}.
```

Comment:
This looks handy, but it seems like a more consistent/specific name would be `get_region_from_metadata`, no?

Reply:
yes, please follow the module naming.